src/custom/pages/Claim/InvestmentFlow/InvestOption.tsx

```diff
@@ -13,79 +11,96 @@ import { ApprovalState } from 'hooks/useApproveCallback'
 import { useCurrencyBalance } from 'state/wallet/hooks'
 import { useActiveWeb3React } from 'hooks/web3'
 import { useClaimDispatchers, useClaimState } from 'state/claim/hooks'
+import { StyledNumericalInput } from 'components/CurrencyInputPanel/CurrencyInputPanelMod'
 
 import { ButtonConfirmed } from 'components/Button'
 import { ButtonSize } from 'theme'
 import Loader from 'components/Loader'
 import { useErrorModal } from 'hooks/useErrorMessageAndModal'
+import { tryParseAmount } from 'state/swap/hooks'
+import { ONE_HUNDRED_PERCENT, ZERO_PERCENT } from 'constants/misc'
+import { PERCENTAGE_PRECISION } from 'constants/index'
 
-const RangeSteps = styled.div`
-  display: flex;
-  align-items: center;
-  width: 100%;
-  justify-content: space-between;
-`
-
-const RangeStep = styled.button`
-  background: none;
-  border: none;
-  font-size: 0.8rem;
-  cursor: pointer;
-  color: ${({ theme }) => theme.primary1};
-  padding: 0;
-`
-
-const INVESTMENT_STEPS = ['0', '25', '50', '75', '100']
-
-function _scaleValue(maxValue: CurrencyAmount<Currency>, value: string) {
-  // parse percent to string, example 25% -> 4 or 50% -> 2
-  const parsedValue = new Fraction(value, '100')
-
-  // divide maxValue with parsed value to get invest amount
-  return maxValue.multiply(parsedValue).asFraction
+enum ErrorMsgs {
+  Initial = 'Insufficient balance to cover input investment amount. Adjust the amount or go back to remove this investment option',
+  Balance = 'Insufficient balance to cover input investment amount, please adjust the input amount',
+  MaxCost = 'Input amount is bigger than available investment amount, please adjust the input amount',
```

Comment:
These messages name could be improved. I think I would do my proposal in a PR and involve Alex

Reply:
Not sure if they are too long, but my suggestions are the following: 

Initial = 'Insufficient balance to cover the selected investment amount. Either modify the investment amount or unselect the investment option to move forward',
Balance = 'Insufficient balance to cover the selected investment amount, please modify the selected amount to move forward ',
MaxCost = 'Selected amount is higher than available investment amount, please modify the input amount to move forward',